### DISCUSSION TITLE
Migrating an app with a carousel from PF 6.2 to 13

### DISCUSSION THREAD

**Discussion by xidbhonaker** [primefaces/community#198] [PrimeFaces] [answered]
I am moving an app from 6.2 to 13.  It contains a carousel compenent, used for scrolling through sponsors.  It scrolls  single ad at a time, and restarts. Here is the definition:

           <p:carousel id="adsCarousel" border="0"
					 value="#{advertisersBean.dataList}" 
					 var="adv"
					 widgetVar="adv_list"
					 numVisible="1"
					 circular="true"
					 autoPlayInterval="3000"
					 effect="fade">
			<a href="#{adv.url}" target="_blank"><img src="#{adv.image}" /></a><br/>
            <h:outputText value="#{adv.displayText}" style="display: block; text-align: center; font-weight:bold; border: 0px;" />
           </p:carousel>

After moving to 13.0, the component had 2 changes... the 'indicator' fields (which I fixed with paginator="false", and a prev and next scroller.  I can't find a way to get rid of these.  If I set the style="overflow: hidden", the framework overrides it to "overflow: flex".    How do I get rid of the arrows and let the graphic show in  its entirety?  

Bill

**Comment by FlipWarthog** [comment, marked as answer]
You can hide those nav buttons with:
```css
body .ui-carousel-prev, body .ui-carousel-next {
    display: none;
}
```

**Reply by FlipWarthog** [reply]
Side note, `autoPlayInterval` in your example is not the right case. It needs to be `autoplayInterval` for it to take effect.

**Comment by xidbhonaker** [comment]
Thanks, FlipWarthog,  I will try that first thing tomorrow.  I did try that but without the body keyword.  And will fix the case of the autoplay property.

**Reply by FlipWarthog** [reply]
Yep, adding body increases the CSS specificity of the style so it applies over the default
